frontend/www/js/omegaup/components/problem/SettingsSummary.test.ts

```diff
@@ -87,4 +87,14 @@ describe('SettingsSummary.vue', () => {
 
     expect(wrapper.find('table').exists()).toBeFalsy();
   });
+
+  it('Should handle empty problem settings summary in problem', () => {
+    const wrapper = mount(problem_SettingsSummary, {
+      propsData: Object.assign({}, baseSettingsSummaryProps, {
+        problem: { languages: [], accepts_submissions: true },
+      }),
```

Comment:
```suggestion
      propsData: {
        ...baseSettingsSummaryProps,
        problem: { languages: [], accepts_submissions: true },
      },
```

aquí podemos usar el nuevo shorthand!

Reply:
Ok, aplico este cambio en el PR #5926